styles.css

```diff
@@ -0,0 +1,541 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+:root {
+  --Primary: #172b4d;
+  --Secondary: #36b37f;
+}
+
+.fa-bars {
+  color: #28352f;
+}
+
+.burger {
+  display: none;
+}
+
+.flex-all {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.flex-center {
+  text-align: center;
+  display: flex;
+  align-items: center;
+}
+
+.flex-new {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.grid {
+  display: grid;
+  align-items: center;
+  justify-content: center;
+}
+
+.widh-marg {
+  max-width: 1020px;
+  margin: auto;
+}
+
+.mb {
+  margin-bottom: 3rem;
+}
+
+.mt {
+  margin-top: 7rem;
+}
+
+* {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+li {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.hero {
+  height: 100vh;
+  background: url(assest/img/bgimg/Header-llustration-desktop.png);
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  position: relative;
+}
+
+.header-content {
+  width: 95%;
+  margin: auto;
+  padding-top: 1.5%;
+  position: fixed;
+}
+
+.logo {
+  visibility: hidden;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.nav-list {
+  display: flex;
+  flex-direction: row;
+  width: 57%;
+  gap: 35px;
+}
+
+.nav-list a {
+  color: var(--Primary);
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 20px;
+}
+
+.nav-list li a:hover {
+  color: var(--Secondary);
+}
+
+.contant {
+  flex-direction: column;
+  gap: 24px;
+  justify-content: center;
+  height: 100vh;
+}
+
+h1 {
+  font-weight: 1300;
+  font-size: 70px;
+  line-height: 70px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: var(--Primary);
+}
+
+h2 {
+  font-weight: 1300;
+  font-size: 70px;
+  line-height: 70px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: var(--Secondary);
+}
+
+p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: var(--Primary);
+  flex: none;
+  flex-grow: 0;
+  max-width: 750px;
+}
+
+.social {
+  gap: 20px;
+  margin-top: 3%;
+}
+
+.next {
+  margin-top: 5%;
+  display: flex;
+}
+
+.head {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: var(--Primary);
+}
```

Comment:
- [ ] Great work using the same `color` and `fonts` as described in Figma. For the header of the _**Works**_ section, kindly follow the structure. On Figma, it is two-layered while on yours it is one-layered. Also, notice the green line under the header. Your design doesn't have it. 👍 💻 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/219485914-5a4dd7d0-f1d8-4abb-9ac6-a7c7d0421830.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/219486198-afaaeda0-26c4-4a17-a78c-f69954ebdac0.png)